src/components/PatientList/PatientListResults/PatientListResultsItem.js

```diff
@@ -0,0 +1,57 @@
+import React, { PropTypes } from 'react';
+import FontAwesome from 'react-fontawesome';
+import moment from 'moment';
+
+const PatientListResultsItem = ({patient}) => {
+	const { gender } = patient;
+	const age = moment().diff(moment(patient.birthDate), 'years');
+	// TODO This isn't a viable way to extract names
+	const name = `${patient.name[0].family}, ${patient.name[0].given}`;
```

Comment:
I think a middleware would help here. Take a look at the middlewares for populations, huddles, and risks for ideas.